pages/release-notes/20181115-1.13.md

```diff
@@ -66,7 +63,6 @@ The keepalived charm can be used to run multiple kube-api-loadbalancers behind a
  - Updated to latest CUDA and removed version pins of nvidia-docker stack ([Issue](https://github.com/juju-solutions/layer-docker/pull/123))
  - Removed portmap from Calico resource ([Issue](https://github.com/juju-solutions/layer-calico/pull/29))
  - Removed CNI bins from flannel resource ([Issue](https://github.com/juju-solutions/layer-canal/pull/25))
- - Reverted kubernetes-master to old (non-CSI) Ceph integration on k8s 1.11, 1.10 ([Issue](https://github.com/juju-solutions/kubernetes/pull/180))
```

Comment:
Removed because this was handled as part of the `Fixed ceph volumes fail to attach with "node has no NodeID annotation"` issue.